src/sensors/ninedof.rs

```diff
@@ -57,9 +58,9 @@ impl Ninedof {
     }
 }
 
-impl Default for Ninedof {
+impl Default for NinedofDriver {
```

Comment:
Why this `impl`? Shouldn't everything be extracted from `Drivers`?

Reply:
fixed.